src/documents/tutorial-prefsFramework/CreatingAPrefsEditor.md

```diff
@@ -0,0 +1,779 @@
+        autoPilot: "${autoPilot}"
+    }
+});
+```
+In this code snippet, the Panel is created using a call to the Infusion Framework function
+`fluid.defaults()`, just as the Primary Schema was. As with the Primary Schema, the call to
+`fluid.defaults()` is passed two arguments:
+1. a string name (`"minEditor.panels.autoPilot"`), and
+2. a JavaScript object containing options for configuring the component – in this case, the Panel.
+
+The screenshot in Figure 2 (above) shows what the Panel looks like to the user: A single checkbox
+and label, with a header above. The options for configuring this Panel
+include four properties: `gradeNames`, `preferenceMap`, `selectors` and `protoTree`:
+<dl>
+<dt>`gradeNames`</dt>
+<dd>As we saw with the Primary Schema, any call to `fluid.defaults()` must refer to any parent
+grades using the `gradeNames` property. Panels must use the `"fluid.prefs.panel"` grade.
+Using this grade automatically buys you a lot of Framework supports necessary for Panels.
+</dd>
+<dt>`preferenceMap`</dt>
+<dd>A Panel must have a _ Preference Map_, which maps the information in the Primary Schema
+into your Panel.Let’s look at this one more closely:
+<pre class="highlight">
+<code class="hljs javascript">{
+preferenceMap: {
+    "minEditor.autoPilot": {
+        "model.autoPilot": "default"
+    }
+},</code></pre>
+
+The first line of the  Preference Map, `“minEditor.autoPilot”`, is the name of the preference.
+This exactly matches the name we saw in the Primary Schema earlier. The value for this
+key is a JavaScript object that defines how this particular preference relates to the Panel’s
+internal data model.
+
+The content of this  Preference Map is a key/value pair:
+<ul>
+<li> The key, `“model.autoPilot”`, is an [EL path](../FrameworkConcepts.md#el-paths) into the
+Panel’s data model. An “EL path” is just a
+dot-separated path built from names. In this case, it means “the `autoPilot`
+property of the [`model`](../FrameworkConcepts.md#model-objects) property” of the Panel.
+<aside class="infusion-docs-callout">
+[Models](../FrameworkConcepts.md#model-objects) are central to Infusion, which,
+while not formally a [Model-View-Controller framework](../FrameworkConcepts.md#mvc),
+embodies the the separation of concerns that is central to MVC.
+Most Infusion components have an internal model, for maintaining the state of the component.
+</aside>
+</li>
+<li> The value, `“default”`, is an EL path referencing the `“default”` property in the Primary Schema.</li>
+</ul>
+This  Preference Map is saying two things:
+<ol>
+<li>The preference called `“minEditor.autoPilot”` should be stored in the Panel’s model
+in a property called `autoPilot`, and</li>
+<li>the initial value for the property should be taken from the `“default”` property
+of the Primary Schema.</li>
+</ul>
+A Preference Map can specify other locations for Primary Schema information, besides the model.
```

Comment:
I wonder if "destinations" might be a better word than "locations" as I'm not sure if you mean the location in the Primary Schema or in the panel.